.github/workflows/alphabetise.yml

```diff
@@ -28,6 +28,8 @@ jobs:
         # Skip main in forks
         if: "github.repository == 'quarkiverse/quarkiverse-docs' && endsWith(github.ref, '/main')"
         run: |
+          git config --global user.name "${GITHUB_ACTOR}"
+          git config --global user.email "${GITHUB_ACTOR}@users.noreply.github.com"          
```

Comment:
I would recommend to use this instead:

https://github.com/quarkusio/quarkusio.github.io/blob/main/.github/workflows/sync-main-doc.yml#L49-L52